src/main/java/net/minecraftforge/fml/common/registry/EntityEntryBuilder.java

```diff
@@ -119,6 +122,35 @@ private EntityEntryBuilder()
         return this;
     }
 
+    /**
+     * Sets the factory of the entity used when constructing an entity sent to a client from a server.
+     *
+     * @param networkSpawnFactory The entity factory, used when being spawned over the network
+     * @return This builder
+     * @throws NullPointerException If {@code networkSpawnFactory} is {@code null}
+     */
+    @Nonnull
+    public final EntityEntryBuilder<E> networkSpawnFactory(@Nonnull final Function<FMLMessage.EntitySpawnMessage, E> networkSpawnFactory)
```

Comment:
All member fields in `FMLMessage.EntitySpawnMessage` have no access modifier (i.e. package private), and have no accessor.
https://github.com/MinecraftForge/MinecraftForge/blob/272b337aa33864ac3d9a855c5cd9ee1f79ac0a8b/src/main/java/net/minecraftforge/fml/common/network/internal/FMLMessage.java#L127-L142
If we have `networkSpawnFactory`, consider making them accessible by outside class(es).